poshc2/client/command_handlers/ImplantHandler.py

```diff
@@ -477,6 +491,49 @@ def do_show_urls(user, command):
     clear()
 
 
+def do_show_hosted_files(user, command):
+    files = update_cache_urls()
+    filesformatted = "ID  URI  FilePath  ContentType  Base64  Active\n"
+    for i in files:
+        filesformatted += "%s  %s  %s  %s  %s %s \n" % (i[0], i[1], i[2], i[3], i[4], i[5])
+    print_good(filesformatted)
+    input("Press Enter to continue...")
+    clear()
+
+
+def do_add_hosted_file(user, command):
+    URI = input("URI Path: downloads/2020/application.docx ")
```

Comment:
Maybe should have an `e.g.` to make it clear it's an example, like the others.
We should also check if the URI starts with a / and add one/trim it depending on what we're expecting.